libraries/chain/webassembly/eos-vm-oc/switch_stack_linux.s

```diff
@@ -0,0 +1,13 @@
+.file	"switch_stack_linux.s"
```

Comment:
Is this really linux only?

Reply:
It's not strictly linux only, but it won't work on OSX.  The code is fine on OSX, but the file format is not.